sys/amd64/linux/linux_proto.h

```diff
@@ -23,25 +23,24 @@ struct proc;
 
 struct thread;
 
-#ifdef CPU_CHERI
-#define	CHERI_PADL_(t)	(sizeof (t) > sizeof(register_t) ? \
-		0 : sizeof(register_t))
-#define	CHERI_PADR_(t)	(sizeof (t) > sizeof(register_t ) ? \
-		0 : sizeof(__intcap_t) - (CHERI_PADL_(t) + sizeof(register_t)))
-#else
-#define	CHERI_PADL_(t)	0
-#define	CHERI_PADR_(t)	0
-#endif
-
-#define	PAD_(t)	(sizeof(register_t) <= sizeof(t) ? \
-		0 : sizeof(register_t) - sizeof(t))
+#define	PAD_(t)	(sizeof(syscallarg_t) <= sizeof(t) ? \
+		0 : sizeof(syscallarg_t) - sizeof(t))
 
 #if BYTE_ORDER == LITTLE_ENDIAN
 #define	PADL_(t)	0
 #define	PADR_(t)	PAD_(t)
+#elif _MIPS_SZCAP == 256
```

Comment:
We should probably remove all remaining uses of _MIPS_SZCAP and replace them with either
`__CHERI_CAPABILITY_WIDTH__` (128/256) or `__SIZEOF_CHERI_CAPABILITY__` (16/32).
But that can be done with a global search replace after this change has landed.